<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QMdiArea" name="mdiArea"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>680</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuList">
    <property name="title">
     <string>&amp;List</string>
    </property>
    <addaction name="action_New"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSearch"/>
    <addaction name="separator"/>
    <addaction name="action_Export_to_PDF"/>
    <addaction name="action_Remove"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="action_Cut"/>
    <addaction name="action_Paste"/>
    <addaction name="separator"/>
    <addaction name="action_Add"/>
    <addaction name="actionDownload"/>
    <addaction name="action_Delete"/>
   </widget>
   <addaction name="menuList"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_New"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSearch"/>
   <addaction name="action_Export_to_PDF"/>
   <addaction name="separator"/>
   <addaction name="action_Cut"/>
   <addaction name="action_Paste"/>
   <addaction name="action_Add"/>
   <addaction name="actionDownload"/>
   <addaction name="action_Delete"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="statusTip">
    <string>Open existing filelist</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>&amp;Sync</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_Cut">
   <property name="text">
    <string>Cu&amp;t</string>
   </property>
   <property name="statusTip">
    <string>Cut file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="action_Copy">
   <property name="text">
    <string>&amp;Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_Paste">
   <property name="text">
    <string>&amp;Paste</string>
   </property>
   <property name="statusTip">
    <string>Paste file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="action_Delete">
   <property name="text">
    <string>&amp;Delete file</string>
   </property>
   <property name="statusTip">
    <string>Delete file</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="action_Server">
   <property name="text">
    <string>&amp;Server</string>
   </property>
  </action>
  <action name="action_Preference">
   <property name="icon">
    <iconset>
     <normaloff>../pic/PNG/calc_24.png</normaloff>../pic/PNG/calc_24.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Preference</string>
   </property>
  </action>
  <action name="action_Add">
   <property name="text">
    <string>&amp;Add</string>
   </property>
   <property name="statusTip">
    <string>Add file</string>
   </property>
  </action>
  <action name="action_Edit">
   <property name="text">
    <string>&amp;Edit</string>
   </property>
  </action>
  <action name="action_Export_to_PDF">
   <property name="text">
    <string>&amp;Export to PDF</string>
   </property>
   <property name="statusTip">
    <string>Export filelist to pdf</string>
   </property>
  </action>
  <action name="action_Remove">
   <property name="text">
    <string>&amp;Remove filelist</string>
   </property>
   <property name="statusTip">
    <string>Remove filelist</string>
   </property>
   <property name="whatsThis">
    <string>You can remove empty filelist</string>
   </property>
  </action>
  <action name="action_New">
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="statusTip">
    <string>Create new filelist</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_Close">
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="statusTip">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSearch">
   <property name="text">
    <string>Search</string>
   </property>
   <property name="statusTip">
    <string>Search</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionDownload">
   <property name="text">
    <string>Download</string>
   </property>
   <property name="statusTip">
    <string>Download file</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
